src/SqlVirtualMachine/SqlVirtualMachine.Test/ScenarioTests/SqlVirtualMachineTestRunner.cs

```diff
@@ -0,0 +1,59 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Xunit.Abstractions;
+using Microsoft.Azure.Commands.TestFx;
+
+namespace Microsoft.Azure.Commands.SqlVirtualMachine.Test.ScenarioTests
+{
+    public class SqlVirtualMachineTestRunner
+    {
+        protected readonly ITestRunner TestRunner;
+
+        protected SqlVirtualMachineTestRunner(ITestOutputHelper output)
+        {
+            TestRunner = TestManager.CreateInstance(output)
+                .WithNewPsScriptFilename($"{GetType().Name}.ps1")
+                .WithProjectSubfolderForTests("ScenarioTests")
+                .WithCommonPsScripts(new[]
+                {
+                    @"Common.ps1",
+                    @"../AzureRM.Resources.ps1",
+                    @"../AzureRM.Storage.ps1"
+                })
+                .WithNewRmModules(helper => new[]
+                {
+                    helper.RMProfileModule,
+                    helper.GetRMModulePath("Az.SqlVirtualMachine.psd1"),
+                    helper.GetRMModulePath("Az.Compute.psd1"),
+                    helper.GetRMModulePath("Az.Network.psd1")
+                })
+                .WithNewRecordMatcherArguments(
```

Comment:
The matcher is PermissiveRecordMatcherWithResourceApiExclusion. Should also invoke WithRecordMatcher method